tools/deploy-docs/build-docs.ts

```diff
@@ -113,6 +178,10 @@ async function addVersionTs(version: Version, versionTsPath: string) {
   await writeFile(versionTsPath, source, 'utf8');
 }
 
+async function createFileWithCommitHash(path: string, projectDir: string) {
```

Comment:
```suggestion
async function addCommitHash(path: string, projectDir: string) {
```
To conform to already existing `addVersionNameToPackageJson` and `addVersionTs`.